raytracing/rays.py

```diff
@@ -712,7 +712,9 @@ def randomRay(self) -> Ray:
         if len(self._rays) == self.maxCount:
             raise AttributeError("Cannot generate more random rays, maximum count achieved")
 
+        np.random.seed(0)
```

Comment:
I don't think this is the way we want to do it. We want to seed the test properly, but not the code itself. So we need to seed beefore testing.  I looked at the code, and I added it to `envtest.RaytracingTestCase` so it will *always* be seeded with 0 for every single test, so we will always get the same sequence of random numbers **when testing**:

```
class RaytracingTestCase(unittest.TestCase):
    tempDir = os.path.join(tempfile.gettempdir(), "tempDir")
    def __init__(self, tests=()):
        super(RaytracingTestCase, self).__init__(tests)

    def setUp(self):
        np.random.seed(0)
```

Reply:
I committed and uploaded, and I removed your `seed()` calls.

Reply:
I removed your seed calls, because they would simply have forced the *same* rays to come out every single time.

Reply:
Click "Resolve conversation" to confirm you have read and understood this, then we could merge?